web/views.py

```
@@ -6485,3 +6487,163 @@ def delete_post(request, post_id):
+def create_voice_chat_room(request):
+    """Create a new voice chat room"""
+    if request.method == "POST":
+        name = request.POST.get("name", f"{request.user.username}'s Room")
+        room = VoiceChatRoom.objects.create(name=name, created_by=request.user)
+        room.participants.add(request.user)
+        return redirect("voice_chat_room", room_id=room.id)
+    return render(request, "web/voice_chat/create.html")
+
+
+@login_required
+def list_voice_chat_rooms(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `list_voice_chat_rooms`

(ANN201)